testdata/python-small/obj/api/google.cloud.texttospeech_v1beta1.types.ListVoicesRequest.yml

```diff
@@ -0,0 +1,30 @@
+### YamlMime:UniversalReference
+api_name: []
+items:
+- children: []
+  class: google.cloud.texttospeech_v1beta1.types.ListVoicesRequest
+  fullName: google.cloud.texttospeech_v1beta1.types.ListVoicesRequest
+  inheritance:
+  - inheritance:
+    - type: builtins.object
+    type: proto.message.Message
+  langs:
+  - python
+  module: google.cloud.texttospeech_v1beta1.types
+  name: ListVoicesRequest
+  source:
+    id: ListVoicesRequest
+    path: google/cloud/texttospeech_v1beta1/types/cloud_tts.py
+    remote:
+      branch: HEAD
+      path: google/cloud/texttospeech_v1beta1/types/cloud_tts.py
+      repo: https://github.com/googleapis/python-texttospeech.git
+    startLine: 61
+  summary: 'The top-level message sent by the client for the `ListVoices`
+
+    method.'
```

Comment:
extra newline?

Reply:
This is auto generated from the Python text to speech repo. I'm using it to test because Python surfaced a couple issues I'd like to improve in the templates.